eng/helix/helix.proj

```diff
@@ -119,12 +123,13 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="Gather" BeforeTargets="Build">
+  <!-- In inner (per-queue) builds, Publish the test projects and create work items. -->
+  <Target Name="Gather" BeforeTargets="BeforeTest" Condition=" '$(HelixTargetQueue)' != '' ">
     <MSBuild Projects="@(ProjectToBuild)"
-              Targets="CreateHelixPayload"
-              SkipNonexistentTargets="true">
+        BuildInParallel="%(ProjectToBuild.BuildInParallel)"
```

Comment:
/fyi this line is meaningless when using RunHelix.ps1 because Arcade's Build.proj provides the default value (`true`). It might result in some speedup of the CI builds however.